SpywareFilter/sections/tracking_servers.txt

```diff
@@ -3316,6 +3316,8 @@
 ||signup-way.com^$third-party
 ||silverpop.com^$third-party
 ||silverpush.co^$third-party
+||simpleanalyticscdn.com^$third-party
```

Comment:
Hello @catleeball
We already have this rule in tracking_servers.txt

Reply:
Bad day #149538

Reply:
It is a pity that Microsoft did not add to this system the option to undo two lines in one commit as Sergei's comment is not a problem in this case.